src/sentry/api/serializers/models/project.py

```diff
@@ -679,16 +678,14 @@ def get_attrs(
         if not self._collapse(LATEST_DEPLOYS_KEY):
             deploys_by_project = self.get_deploys_by_project(item_list)
 
-        with sentry_sdk.start_span(op="project_summary_serializer.get_lpq_projects"):
-            lpq_projects = realtime_metrics.get_lpq_projects()
         for item in item_list:
             attrs[item]["latest_release"] = latest_release_versions.get(item.id)
             attrs[item]["environments"] = environments_by_project.get(item.id, [])
             attrs[item]["has_user_reports"] = item.id in projects_with_user_reports
             if not self._collapse(LATEST_DEPLOYS_KEY):
                 attrs[item]["deploys"] = deploys_by_project.get(item.id)
-            attrs[item]["symbolication_degraded"] = should_demote_symbolication(
-                project_id=item.id, lpq_projects=lpq_projects, emit_metrics=False
+            attrs[item]["symbolication_degraded"] = realtime_metrics.is_lpq_project(
```

Comment:
tbh, we shouldn’t expose this to customers, as this just creates confusion and customer support requests. but I don’t think this decision is mine to make

Reply:
Yeah I started ripping this out and realized it had ripple effects all over the place, so I left it alone.